src/realmGeolBasin.ttl

```diff
@@ -39,50 +40,54 @@ sorelph:hasPlanetaryStructure rdf:type owl:ObjectProperty .
 
 ###  http://sweetontology.net/realmGeolBasin/BackArcBasin
 soreagb:BackArcBasin rdf:type owl:Class ;
-                   rdfs:subClassOf soreagb:Basin ;
+                   rdfs:subClassOf soreagb:SedimentaryBasin ;
                    rdfs:label "back arc basin"@en .
 
 
-###  http://sweetontology.net/realmGeolBasin/Basin
-soreagb:Basin rdf:type owl:Class ;
+###  http://sweetontology.net/realmGeolBasin/SedimentaryBasin
+soreagb:SedimentaryBasin rdf:type owl:Class ;
```

Comment:
As @cmungall points out, probably best not to change the URI for soreagb:Basin, just change the label to disambiguate better from soreahb:Basin.  The cows have left the barn at this point using opaque URIs in SWEET....

Reply:
OK